### PR TITLE
add new company - UPLERS

### PR DESCRIPTION
**Pull request by janeesh27**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [ x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x ] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x ] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [ x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [ x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [ x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -671,6 +671,7 @@ Name | Website | Region
 [Udacity](/company-profiles/udacity.md) | https://www.udacity.com/ | Worldwide
 [Uhuru](/company-profiles/uhuru.md) | https://uhurunetwork.com/ | Worldwide
 [Uncapped](/company-profiles/uncapped.md) | https://weareuncapped.com/ | Europe, USA
+[Uplers](/company-profiles/uplers.md) | https://www.uplers.com/ | Worldwide
```
Comment (KieranRobson) [inline README.md:674]: Although you may hire globally (?), 

- You wrote India within the region of the profile,
- The user is greeted with `Opening Global Doors for INDIA’S TOP TALENT` when they open the website 
- Your office locations are just listed at `India`
- and finally you state `with the best remote Indian talent to meet their hiring needs.` within the blurb.

For the above reasons, could you change `Worldwide` to `India`

File: company-profiles/uplers.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Uplers
+
+## Company blurb
+
+Uplers is on a mission to provide companies all over the globe with the best remote Indian talent to meet their hiring needs.
```
Comment (KieranRobson) [inline company-profiles/uplers.md:5]: Although you may be on a ` mission to provide companies all over the globe with the best remote Indian talent to meet their hiring needs.`, this doesnt say what the company does. Could you add some extra detail?

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Hi @janeesh27, Thanks for your contribution!

Just two quick suggestions!

edit:

`Uplers Talent Solutions gets you a job opportunity as easily as you develop websites.` Is Uplers a company who people use to find remote jobs, or is Uplers a company who hires remotely, the website isnt super clear about this?

File: README.md (hunk 1 above)
Comment (janeesh27) [reply README.md]: requested information provided

**Review by janeesh27 — COMMENTED** [review]
more information added about the company as requested

**Comment by KieranRobson** [comment]
Thanks @janeesh27, Could you clarify the below comment:

Uplers Talent Solutions gets you a job opportunity as easily as you develop websites. Is Uplers a company who people use to find remote jobs, or is Uplers a company who hires remotely, the website isn't super clear about this?

**Comment by dougaitken** [comment]
Thanks for clarifying @janeesh27 

This adding won't be accepted as the company doesn't meet the criteria as mentioned at the 4th checklist item in the PR template. This company doesn't hire directly, it sounds basically like a match making services.

**Pull request closed without merging**